src/components/TaskRunDetails.vue

```diff
@@ -57,28 +65,41 @@
   import { PKeyValue, PTags } from '@prefecthq/prefect-design'
   import DurationIconText from '@/components/DurationIconText.vue'
   import FlowRunIconText from '@/components/FlowRunIconText.vue'
+  import { useTaskRunResult } from '@/compositions'
+  import { localization } from '@/localization'
   import { TaskRun } from '@/models/TaskRun'
   import { formatDateTimeNumeric } from '@/utilities/dates'
   import { secondsToApproximateString } from '@/utilities/seconds'
 
-  defineProps<{
+  const props = defineProps<{
     taskRun: TaskRun,
     alternate?: boolean,
   }>()
+
+  const { result } = useTaskRunResult(props.taskRun.id)
 </script>
 
 <style>
-  .task-run-details {
-    @apply
-    flex
-    flex-col
-    gap-3
-    items-start
-  }
-
-  .task-run-details__tags {
-    @apply
-    mb-1
-    mr-1
-  }
+.task-run-details { @apply
+  flex
+  flex-col
+  gap-3
+  items-start
+}
+
+.task-run-details__tags { @apply
+  mb-1
+  mr-1
```

Comment:
Could grid styling be used instead to avoid hanging margins? I realize you didn't actually change this, so nbd.

Reply:
I didn't test this without since these styles existed before the PR (they had inconsistent indentation) - @brandonreid feel free to PR what you had in mind there if you feel strongly but I don't want to experiment with that in this PR